aslam_offline_calibration/kalibr/python/kalibr_visualize_distortion

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+import argparse
+import kalibr_common as kc
+import math
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import numpy as np
+import sm
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description=
+        'Visualizes a camchain-cam.yaml distortion.')
+    parser.add_argument(
+        '--cam',
+        dest='chain_yaml',
+        help=
+        'Camera configuration as yaml file (can take multiple configurations).',
+        nargs='+',
+        required=True)
+    parser.add_argument(
+        '--number_of_rectangles',
+        help="Number of rectangles to draw.",
+        default=5)
+    parser.add_argument(
+        '--line_thickness',
+        help="Line thickness.",
+        default=3)
+    parsed_args = parser.parse_args()
+    return parsed_args
+
+def normalize((u,v), focal_lengths, principal_point):
+    x = (u - principal_point[1]) / focal_lengths[1]
```

Comment:
are these indices right? (i.e. should index 0 not refer to the x coordinate?)

Reply:
Pretty sure they are.